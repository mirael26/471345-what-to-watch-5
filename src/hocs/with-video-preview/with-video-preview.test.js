import React from "react";
import renderer from "react-test-renderer";

import withVideoPreview from "./with-video-preview";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withVideoPreview(MockComponent);

it(`Should withVideoPreview render correctly`, () => {
  const noop = () => {};
  const tree = renderer
    .create(
        <MockComponentWrapped
          renderPlayer={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
