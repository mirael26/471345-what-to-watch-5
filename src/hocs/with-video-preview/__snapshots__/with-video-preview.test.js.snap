// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should withVideoPreview render correctly 1`] = `<div />`;
